### PR TITLE
Update assign-licenses-to-user-accounts-with-microsoft-365-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added another cmdlet to pull all synced unlicensed users

### CODE REVIEW

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9268

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell.md [hunk 1]
```diff
@@ -62,6 +62,11 @@ To find the unlicensed accounts in your organization, run this command.
 Get-MgUser -Filter 'assignedLicenses/$count eq 0' -ConsistencyLevel eventual -CountVariable unlicensedUserCount -All
 ```
 
+To find the unlicensed Synchornized users in your oganization, run this command.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/assign-licenses-to-user-accounts-with-microsoft-365-powershell.md]:
```suggestion
To find the unlicensed synchronized users in your organization, run this command.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**